config.ru

```diff
@@ -1,4 +1,6 @@
 # This file is used by Rack-based servers to start the application.
 
-require ::File.expand_path('../config/environment',  __FILE__)
+require_relative "config/environment"
+
 run Rails.application
+Rails.application.load_server
```

Comment:
Not sure it's a good idea to add this... but this is the new default content of `config.ru`:
https://github.com/rails/rails/commit/868866c1fd7f6c79fd48d90febb1fd02a83380e1

Reply:
Not sure what you mean by manually adding servers.

Reply:
@jlledom :point_up: ?

I don't know what this line does. It's just coming in the default templates now.

Reply:
@mayorova @akostadinov this line runs whatever blocks you added through `Rails::Railtie.server`. Check https://api.rubyonrails.org/v6.1.7.7/classes/Rails/Railtie.html, at the section **Run another program when the Rails server starts**.

AFAIK we don't use this feature so there's no difference having this line or not.

Reply:
There might be gems that use, now or in the future?

Reply:
No idea, but it seems to me the best decision is to leave this line. If server are added, this line is required; if no servers are added, this line does nothing. So it will work in all scenarios, and it's only executed once during startup, so it has no impact in performance.